commands/get-docker-tag/get_docker_tag.go

```diff
@@ -0,0 +1,124 @@
+package getdockertag
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/go-git/go-git/v5"
+	"github.com/go-git/go-git/v5/plumbing"
+	"github.com/go-git/go-git/v5/plumbing/storer"
+	"github.com/kurtosis-tech/stacktrace"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"os"
+	"path"
+	"sort"
+	"strings"
+)
+
+const (
+	gitDirname         = ".git"
+	masterBranchName   = "master"
+	dirtySuffix        = "-dirty"
+	getDockerTagCmdStr = "get-docker-tag"
+)
+
+var GetDockerTagCmd = &cobra.Command{
+	Use:   getDockerTagCmdStr,
+	Short: "Prints the expected docker tag given the current state of the Kurtosis repo this command is run on.",
+	RunE:  run,
+}
+
+func run(cmd *cobra.Command, args []string) error {
+	logrus.Infof("Retrieving git information...")
+	currentWorkingDirpath, err := os.Getwd()
+	if err != nil {
+		return stacktrace.Propagate(err, "An error occurred getting the current working directory.")
+	}
+	gitDirpath := path.Join(currentWorkingDirpath, gitDirname)
+	if _, err := os.Stat(gitDirpath); err != nil {
+		if os.IsNotExist(err) {
+			return stacktrace.Propagate(err, "An error occurred getting the git repository in this directory. This means that this binary is not being run from root of a git repository.")
+		}
+	}
+	repository, err := git.PlainOpen(currentWorkingDirpath)
+	if err != nil {
+		return stacktrace.Propagate(err, "An error occurred while attempting to open the existing git repository.")
+	}
+
+	// Determines if working tree is clean
+	appendDirtySuffix := false
```

Comment:
bool variable names be questions (e.g. `shouldAppendDirtySuffix`)

Reply:
gotcha